third_party/bazel/cuda_static/cuda_static.BUILD.tpl

```diff
@@ -21,3 +21,8 @@ cc_import(
     name = "cudnn_static",
     static_library = "lib64/libcudnn_static.a",
 )
+
+cc_import(
+    name = "nvrtc",
+    shared_library = "lib64/libnvrtc.so",
```

Comment:
an shared lib in a build file named static...

Reply:
I did not find a better for this supplementary cuda rule, may cuda_supplement?

Reply:
done